packages/common/container-definitions/src/runtime.ts

```diff
@@ -163,7 +163,7 @@ export interface IContainerContext {
 	readonly closeFn: (error?: ICriticalContainerError) => void;
 	readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
 	readonly quorum: IQuorumClients;
-	readonly audience: IAudience | undefined;
+	readonly audience: IAudience;
```

Comment:
Another bit for the changeset.

Reply:
I'm not sure. Changesets should be useful. If we keep putting every change like that, signal-to-noise ratio would be very low. While someone could implement that interface (for testing?), container runtime always required implementation here. I've checked - Loop does not implement that interface. Even we do not (in our UTs - we implement partial mocks only).

Reply:
Fair pushback. If we confirmed as much as possible that there are no implementations outside the repo, I can see skipping a changeset for this one given its @alpha marker, and the fact that we know it was never intended for consumers to implement so the chance is pretty low. I just err on the side of "let's point out all the _technically_ necessary ones and discuss which ones can be skipped", since we've failed to provide them in the past.